tsl/test/isolation/expected/continuous_aggs_multi.out

```diff
@@ -0,0 +1,59 @@
+INFO:  materializing continuous aggregate public.continuous_view_1: no new range to materialize
+step Refresh1: REFRESH MATERIALIZED VIEW continuous_view_1;
+INFO:  new materialization range not found for public.ts_continuous_test (time column time): no new data
+INFO:  materializing continuous aggregate public.continuous_view_2: no new range to materialize
+step Refresh2: REFRESH MATERIALIZED VIEW continuous_view_2;
+step Refresh1_sel: select * from continuous_view_1 where bkt = 0;
+bkt            cnt            
+
+0              16             
+step Refresh2_sel: select * from continuous_view_2 where bkt = 0;
+bkt            maxl           
+
+0              100            
+
+starting permutation: Refresh1 Refresh2 I1 Refresh2 Refresh2_sel Refresh1 Refresh1_sel
+INFO:  new materialization range for public.ts_continuous_test (time column time) (30)
+INFO:  materializing continuous aggregate public.continuous_view_1: new range up to 30
+step Refresh1: REFRESH MATERIALIZED VIEW continuous_view_1;
+INFO:  new materialization range for public.ts_continuous_test (time column time) (35)
+INFO:  materializing continuous aggregate public.continuous_view_2: new range up to 35
+step Refresh2: REFRESH MATERIALIZED VIEW continuous_view_2;
+step I1: INSERT INTO ts_continuous_test SELECT 0, i*10 FROM (SELECT generate_series(0, 10) AS i) AS i;
+INFO:  new materialization range not found for public.ts_continuous_test (time column time): no new data
+INFO:  materializing continuous aggregate public.continuous_view_2: no new range to materialize
+step Refresh2: REFRESH MATERIALIZED VIEW continuous_view_2;
+step Refresh2_sel: select * from continuous_view_2 where bkt = 0;
+bkt            maxl           
+
+0              100            
+INFO:  new materialization range not found for public.ts_continuous_test (time column time): no new data
+INFO:  materializing continuous aggregate public.continuous_view_1: no new range to materialize
+step Refresh1: REFRESH MATERIALIZED VIEW continuous_view_1;
+step Refresh1_sel: select * from continuous_view_1 where bkt = 0;
+bkt            cnt            
+
+0              16             
+
+starting permutation: LockCompleted Refresh1 Refresh2 UnlockCompleted
+step LockCompleted: BEGIN; LOCK TABLE _timescaledb_catalog.continuous_aggs_completed_threshold IN SHARE MODE;
+INFO:  new materialization range for public.ts_continuous_test (time column time) (30)
+INFO:  materializing continuous aggregate public.continuous_view_1: new range up to 30
+step Refresh1: REFRESH MATERIALIZED VIEW continuous_view_1; <waiting ...>
+INFO:  new materialization range for public.ts_continuous_test (time column time) (35)
+INFO:  materializing continuous aggregate public.continuous_view_2: new range up to 35
+step Refresh2: REFRESH MATERIALIZED VIEW continuous_view_2; <waiting ...>
+step UnlockCompleted: ROLLBACK;
+step Refresh1: <... completed>
```

Comment:
@JLockerman does this test actually make sure the last transaction runs concurrently? Might they still be serialized in their completion? Not sure how to resolve this though.

Reply:
@cevian This guarantees they both complete, but not that they do so concurrently. To check that I think we'd need to block `Refresh1` on some lock that `Refresh2` does not take.